.github/workflows/14-backend-pitest.yml

```
@@ -30,7 +30,7 @@ jobs:
     # branch each time the main branch changes
 
     - name: Pitest
-      run: mvn test org.pitest:pitest-maven:mutationCoverage -DmutationThreshold=100 
+      run: run: mvn pitest:mutationCoverage -DmutationThreshold=100
```

You have `run: run:` where I think you just want: `run: `
```suggestion
      run: mvn pitest:mutationCoverage -DmutationThreshold=100
```